repos/system_upgrade/common/actors/copydnfconfintotargetuserspace/libraries/copydnfconfintotargetuserspace.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from leapp.libraries.stdlib import api
+from leapp.models import CopyFile, TargetUserSpacePreupgradeTasks
+
+
+def process():
+    src = "/etc/dnf/dnf.conf"
+    if os.path.exists("/etc/leapp/files/dnf.conf"):
+        src = "/etc/leapp/files/dnf.conf"
+
+    api.current_logger().debug(
+        "Copying dnf.conf at {} to the target userspace".format(src)
+    )
```

Comment:
@matejmatuska note this could be a little bit confusing as this actor is not copying anything. it just creates a msg, that will be processed by the other actor that will do the operation. same about the docstring of the actor. but it's a minor issue and we keep it as that. just saying to help you with writting of future docstrings.